gst/nnstreamer/elements/gsttensor_decoder.c

```diff
@@ -515,8 +516,9 @@ gst_tensordec_set_property (GObject * object, guint prop_id,
     }
     case PROP_CONFIG:
     {
-      const gchar *config_path = g_value_get_string (value);
-      gst_tensor_parse_config_file (config_path, object);
+      g_free (self->config_path);
+      self->config_path = g_strdup (g_value_get_string (value));
+      gst_tensor_parse_config_file (self->config_path, object);
```

Comment:
May check success/failure of `gst_tensor_parse_config_file`, changing return value of the method.

Reply:
it returns nothing. (void, not bool)